phpstan.neon.dist

```diff
@@ -128,3 +128,8 @@ parameters:
 		-
 			message: "#Call to function method_exists\\(\\) with ApiPlatform\\\\Core\\\\JsonApi\\\\Serializer\\\\ItemNormalizer and 'setCircularReferenc…' will always evaluate to false\\.#"
 			path: tests/Core/JsonApi/Serializer/ItemNormalizerTest.php
+
+		# Expected, due to maker fixtures php version compatibility
+		-
+			message: "#Method App\\\\State\\\\CustomStateProvider::provide\\(\\) should return array\\|object\\|null but return statement is missing\\.#"
+			path: tests/Fixtures/Symfony/Maker/CustomStateProvider.php
```

Comment:
may I can add conditionally the php8 return type? 

maker has a `PhpCompatUtil` class but it does not expose its `getPhpVersion()` method :/

Reply:
Maybe could we copy it temporarily? cc @weaverryan

Reply:
IMO this method should be public in `PhpCompatUtil`
If @weaverryan is OK, I may provide a PR in the maker and in the same time temporarily copy it here until it is released in the maker

Reply:
(or maybe expose a variable to all the templates: `$is_php8` - but this is a discussion we should have in the maker repository)